src/settingsScreen.js

```diff
@@ -434,7 +434,11 @@ class SettingsScreen {
             for (let index = 0; index < sensitivities.length; index++) {
                 const sensitivity = sensitivities[index];
                 await match.accessible.ButtonBarButton(text.general.Add).tap();
-                await match.accessible.Label(`${sensitivity.bgValuePerInsulinUnit} ${unitsSuffix}`).atIndex(1).tap();
+                let currentMax = 500;
```

Comment:
[nit] Consider moving this constant to a "loop constants" file so as these things change there is one place for them all to update.

Reply:
good idea